packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -16,53 +19,73 @@ class CeramicDaemon {
       next()
     })
     this.app.post(toApiPath('/create'), this.createDoc.bind(this))
-    this.app.get(toApiPath('/show/ceramic/:doctype/:cid'), this.show.bind(this))
-    this.app.get(toApiPath('/state/ceramic/:doctype/:cid'), this.state.bind(this))
-    this.app.post(toApiPath('/change/ceramic/:doctype/:cid'), this.change.bind(this))
+    this.app.get(toApiPath('/show/ceramic/:cid'), this.show.bind(this))
+    this.app.get(toApiPath('/state/ceramic/:cid'), this.state.bind(this))
+    this.app.post(toApiPath('/change/ceramic/:cid'), this.change.bind(this))
     const server = this.app.listen(7007, () => {
       console.log('Ceramic API running on port 7007')
+      console.log('User DID:', ceramic.user.DID)
     })
     server.keepAliveTimeout = 60 * 1000
   }
 
   static async create (ipfsHost: string = IPFS_HOST) {
     const ipfs = ipfsClient(ipfsHost)
     const ceramic = await Ceramic.create(ipfs)
+    const idWallet = new IdentityWallet(async () => true, { seed })
+    await ceramic.setDIDProvider(idWallet.get3idProvider())
     return new CeramicDaemon(ceramic)
   }
 
   async createDoc (req: Request, res: Response, next: NextFunction) {
     const { doctype, genesis, onlyGenesis } = req.body
     if (!doctype || !genesis) {} // TODO - reject somehow
-    const doc = await this.ceramic.createDocument(genesis, doctype, { onlyGenesis })
-    res.json({ docId: doc.id, state: doc.state })
+    try {
+      const doc = await this.ceramic.createDocument(genesis, doctype, { onlyGenesis })
+      res.json({ docId: doc.id, state: doc.state })
+    } catch (e) {
+      res.json({ error: e.message })
```

Comment:
Do we need to handle `Response` error status codes?

Reply:
Yeah, right now this isn't really using best practices for express or rest apis in general. We should make sure to do this properly!